GeneratorInterface/Pythia6Interface/src/Pythia6Service.cc

```diff
@@ -382,11 +382,18 @@ void Pythia6Service::setPYUPDAParams(bool afterPyinit)
 
 void Pythia6Service::setSLHAFromHeader( const std::vector<std::string> &lines )
 {
-
 	std::set<std::string> blocks;
 	unsigned int model = 0, subModel = 0;
+        bool useboost = true;
```

Comment:
NAK. This is really not needed. This is a dead code which will be optimised out. Remove it. Stick with simple:
```
fnamest = std::string(boost::filesystem::unique_path().string());
fname = fnamest.c_str();
```
Then verify that it works (by running your workflows, valgrind, etc.)